.github/workflows/main-prs.yml

```diff
@@ -0,0 +1,26 @@
+name: move PR to main
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
I think you'll want to restrict this to `types: [ opened ]`, otherwise it'll get triggered on every type of (pull request even)[https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request].

Reply:
Yeah I had it like that so that it would catch someone editing the base _away_ from main to master, but you're right that we don't need to trigger this a bajillion times.